util/helm.go

```diff
@@ -122,3 +122,8 @@ func CopyDir(src, dst string) error {
 	filepath.Walk(src, walker)
 	return failure
 }
+
+// ExpandChartPath - Expand chart path from name and home directory
+func ExpandChartPath(dir, chartName string) string {
```

Comment:
I was just looking into that.  We also have the path const defined `WorkspacePath`, `WorkspaceChartPath`, etc which do not respect the config setting